gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -339,7 +339,8 @@ private ManagedChannel createSingleChannel() throws IOException, GeneralSecurity
       isDirectPathXdsEnabled = Boolean.parseBoolean(envProvider.getenv(DIRECT_PATH_ENV_ENABLE_XDS));
       if (isDirectPathXdsEnabled) {
         // google-c2p resolver target must not have a port number
```

Comment:
Should the comment be updated too?

Reply:
Thanks for asking. I think the comment should be fine as we will eventually use `google-c2p` once `google-c2p-experimental` is stable.